packages/rspack-cli/package.json

```diff
@@ -26,27 +26,23 @@
     "directory": "packages/rspack-cli"
   },
   "devDependencies": {
+    "@types/rechoir": "^0.6.1",
     "@types/webpack-bundle-analyzer": "^4.6.0",
     "concat-stream": "^2.0.0",
     "execa": "^5.0.0",
     "internal-ip": "6.2.0",
     "source-map-support": "^0.5.19",
     "ts-node": "10.9.1"
   },
-  "peerDependencies": {
-    "ts-node": ">= 10"
-  },
-  "peerDependenciesMeta": {
-    "ts-node": {
-      "optional": true
-    }
-  },
   "dependencies": {
     "@discoveryjs/json-ext": "^0.5.7",
     "@rspack/core": "workspace:*",
     "@rspack/dev-server": "workspace:*",
     "colorette": "2.0.19",
+    "interpret": "^3.1.1",
+    "rechoir": "^0.8.0",
     "semver": "6.3.0",
+    "type-fest": "^2.14.0",
```

Comment:
this dependency is unnecessary we can write our own type

Reply:
> this dependency is unnecessary we can write our own type

Looking at the type-fest source code, packageJson actually has a large number of type definitions. Is it necessary to copy the complete type definition, or simply write it as {"type": "module"}

Reply:
write type: module is enough for our case